### PR TITLE
Update get_certificate.py

### PR DESCRIPTION
**Pull request by jimwitte**
##### SUMMARY
provide example to calculate number of days until cert expires from get_certificate result.

##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

##### SUMMARY

provide example to calculate number of days until cert expires from get_certificate result.

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

Calculating how long before a cert expires would seem to be a common task using the output of this module, but took me a while to figure out how to work with the datetime format produced by the module, so wanted to share a solution.

```paste below

```

### CODE REVIEW

**Comment by acozine** [comment]
Thanks @jimwitte for contributing to the Ansible documentation. Your PR currently has two small flaws standing between it and a clean CI run:

```
lib/ansible/modules/crypto/get_certificate.py:110:1: W293 blank line contains whitespace
lib/ansible/modules/crypto/get_certificate.py:115:161: E501 line too long (165 > 160 characters)
```

Can you fix those two issues?

File: lib/ansible/modules/crypto/get_certificate.py [hunk 1]
```diff
@@ -107,6 +107,12 @@
   delegate_to: localhost
   run_once: true
   register: cert
+  
+- name: how many days until cert expires
```
Comment (felixfontein) [inline lib/ansible/modules/crypto/get_certificate.py]:
```suggestion
- name: How many days until cert expires
```

**Comment by acozine** [comment]
Thanks @jimwitte for the example and @felixfontein for the review.

**Review by felixfontein — APPROVED** [review]
shipit

**Comment by felixfontein** [comment]
@jwitte @acozine thanks for your work on this! I've created a backport to stable-2.8: #56758

**Pull request merged**